drivers/gpio/gpio_pcal6416a.c

```diff
@@ -0,0 +1,269 @@
+/*
+ * Copyright (c) 2023 SILA Embedded Solutions GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT nxp_pcal6416a
+
+#include <zephyr/logging/log.h>
+
+#include "gpio_pcal64xxa.h"
+
+LOG_MODULE_REGISTER(pcal6416a, CONFIG_GPIO_LOG_LEVEL);
+
+enum {
```

Comment:
Please add `PCAL6408A` register enumerations in this file as well.

Reply:
done